### ISSUE TITLE
Fatal error: Indent.rename

### ISSUE THREAD

**Issue by smimram** [ocaml/merlin#976]
Hi,

On Emacs with company, when I type `List.` and wait for completion I get the following error
```
Company: backend merlin-company-backend error "unknown answer: "exception":"Misc.Fatal_error(\"Ident.rename t\", _)
Raised at file \"src/ocaml/utils/misc.ml\", line 32, characters 2-54
Called from file \"src/ocaml/typing/408/printtyp.ml\", line 1388, characters 41-58
Called from file \"src/ocaml/typing/408/printtyp.ml\", line 1362, characters 36-46
Called from file \"src/analysis/completion.ml\", line 59, characters 14-75
Called from file \"src/frontend/query_commands.ml\", line 38, characters 12-45
Called from file \"src/analysis/completion.ml\", line 152, characters 21-33
Called from file \"list.ml\", line 103, characters 22-25
Called from file \"src/frontend/query_commands.ml\", line 51, characters 18-75
Called from file \"src/ocaml/utils/misc.ml\", line 37, characters 8-15
Re-raised at file \"src/ocaml/utils/misc.ml\", line 49, characters 10-24
Called from file \"src/utils/std.ml\", line 679, characters 8-12
Re-raised at file \"src/utils/std.ml\", line 681, characters 30-39
Called from file \"src/frontend/query_commands.ml\", line 385, characters 6-196
Called from file \"src/utils/local_store.ml\", line 29, characters 8-12
Re-raised at file \"src/utils/local_store.ml\", line 37, characters 4-15
Called from file \"src/kernel/mocaml.ml\", line 40, characters 8-38
Re-raised at file \"src/kernel/mocaml.ml\", line 48, characters 4-15
Called from file \"src/frontend/new/new_commands.ml\", line 65, characters 15-53
Called from file \"src/utils/std.ml\", line 679, characters 8-12
Re-raised at file \"src/utils/std.ml\", line 681, characters 30-39
Called from file \"src/ocaml/utils/misc.ml\", line 37, characters 8-15
Re-raised at file \"src/ocaml/utils/misc.ml\", line 49, characters 10-24
Called from file \"src/frontend/new/new_merlin.ml\", line 99, characters 18-54
"" with args (candidates List.)
```
However, if I type `List.a` I correctly get `assoc` and other completions.

**Comment by Wilfred** [comment]
I'm experiencing this too. Given the ocaml file:

```
type foo = Foo
```

I get a similar exception:

```
$ cat foo.ml| ocamlmerlin single complete-prefix -position 1:0
{"class":"exception","value":"Misc.Fatal_error(\"Ident.rename unit\", _)\nRaised at file \"src/ocaml/utils/misc.ml\", line 32, characters 2-54\nCalled from file \"src/ocaml/typing/408/printtyp.ml\", line 1388, characters 41-58\nCalled from file \"src/ocaml/typing/408/printtyp.ml\", line 1362, characters 36-46\nCalled from file \"src/analysis/completion.ml\", line 59, characters 14-75\nCalled from file \"src/frontend/query_commands.ml\", line 38, characters 12-45\nCalled from file \"src/analysis/completion.ml\", line 152, characters 21-33\nCalled from file \"list.ml\", line 103, characters 22-25\nCalled from file \"src/frontend/query_commands.ml\", line 51, characters 18-75\nCalled from file \"src/ocaml/utils/misc.ml\", line 37, characters 8-15\nRe-raised at file \"src/ocaml/utils/misc.ml\", line 49, characters 10-24\nCalled from file \"src/utils/std.ml\", line 679, characters 8-12\nRe-raised at file \"src/utils/std.ml\", line 681, characters 30-39\nCalled from file \"src/frontend/query_commands.ml\", line 385, characters 6-196\nCalled from file \"src/utils/local_store.ml\", line 29, characters 8-12\nRe-raised at file \"src/utils/local_store.ml\", line 37, characters 4-15\nCalled from file \"src/kernel/mocaml.ml\", line 40, characters 8-38\nRe-raised at file \"src/kernel/mocaml.ml\", line 48, characters 4-15\nCalled from file \"src/frontend/new/new_commands.ml\", line 65, characters 15-53\nCalled from file \"src/utils/std.ml\", line 679, characters 8-12\nRe-raised at file \"src/utils/std.ml\", line 681, characters 30-39\nCalled from file \"src/ocaml/utils/misc.ml\", line 37, characters 8-15\nRe-raised at file \"src/ocaml/utils/misc.ml\", line 49, characters 10-24\nCalled from file \"src/frontend/new/new_merlin.ml\", line 99, characters 18-54\n","notifications":[],"timing":{"clock":5,"cpu":5,"query":3,"pp":0,"reader":0,"ppx":0,"typer":1,"error":0}}
/tmp
```

Looks like it's fixed by #991.

**Comment by let-def** [comment]
Yes, it should be fixed in version 3.3.2 which has been submitted to opam today :).

**Comment by trefis** [comment]
And it has now reached opam.
